docs/kernel-policy.md

```diff
@@ -1,21 +1,35 @@
 # Firecracker's Kernel Support Policy
 
-Once officially supported kernel versions are supported for a **minimum of 2
-years**.
-
-We are validating the currently supported Firecracker releases as per
-[Firecracker’s release policy](../docs/RELEASE_POLICY.md). Starting with release
-`v1.0` each major and minor release will specify the supported kernel versions.
-Adding support for a new kernel version will result in a Firecracker release
-only if compatibility changes are required.
+Firecracker is tightly coupled with the guest and host kernels on which it is
+run. This document presents our kernel support policy which aims to help our
+customers choose host and guest OS configuration, and predict future kernel
+related changes.
+
+We are continuously validating the currently supported Firecracker releases (as
+per [Firecracker’s release policy](../docs/RELEASE_POLICY.md)) using a
+combination of: host linux kernel versions 4.14, 5.10, and 6.1; and guest linux
+kernel versions 4.14, and 5.10. While other versions and other kernel configs
+might work, they are not periodically validated in our test suite, and using
+them might result in unexpected behaviour. Starting with release `v1.0` each
+major and minor release will specify the supported kernel versions.
+
+Once a kernel version is officially enabled, it is supported for a **minimum of
+2 years**. Adding support for a new kernel version will result in a Firecracker
+release only if compatibility changes are required.
+
+| Host kernel | Guest kernel v4.14 | Guest kernel v5.10 | Min. end of support |
+| ----------: | :----------------: | :----------------: | ------------------: |
+|       v4.14 |          Y         |          Y         |          2021-01-22 |
+|       v5.10 |          Y         |          Y         |          2024-01-31 |
+|        v6.1 |          Y         |          Y         |          2025-10-12 |
 
 The guest kernel configs used in our validation pipelines can be found
 [here](../resources/guest_configs/) while a breakdown of the relevant guest
 kernel modules can be found in the next section.
 
 ## Guest kernel configuration items
 
-Some configuration items that may be of interest are:
+The configuration items that may be relevant for Firecracker are:
```

Comment:
I started wondering about purpose of this section. Is this list supposed to be exhaustive?

Reply:
Those are guestOS configuration so by definition can not be exhaustive because depends also on what the Firecracker user needs in the guest.
Those are a set of configuration we need when building the guestOS to work in conjunction with Firecracker features, and they make sense and we should definitely keep them. This are important info that explain, if you want to use for example virtio-rng device please make sure you build your kernel with `CONFIG_HW_RANDOM_VIRTIO` etc. etc.

Reply:
yeah, this is out of scope of the PR, but we may need to think how we make sure it's maintained...

Reply:
Yes that is true, but we do not add a functionality every day, and when we do we are conscious of it so reviewers and author of  PR can add add whatever configuration is needed in this section (like we did for example with the recent virtio-rng device). We need to pay attentions to details.